src/Tooling/Bit.Tooling.Bswup/README.md

```diff
@@ -1,7 +1,90 @@
 ## Bit Blazor Serviceworker Update Progress (BitBswup)
 
-in order to use BitBswup, please follow these steps:
+to use BitBswup, please follow these steps:
 
-1- install the `Bit.Tooling.Bswup` nuget package
-2- add the requied references to the default document based on the sample in the `index.html` file of the demo project in this repo
-3- configure additional settings in the service worker file (based on the sample shown in the `service-worker.js` file of the demo project)
\ No newline at end of file
+1.Install the `Bit.Tooling.Bswup` nuget package
+2. Disable static file caching.You can follow below code in `Startup.cs` file
+```csharp
+
+public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
```

Comment:
Please remove Configure method. app.UseStaticFiles is enough. They might be using new .net 6 style project which uses web app builder